### ISSUE TITLE
[Quest][Classic] Unfinished Gordok Business

### ISSUE THREAD

**Issue by Braniacz** [dalaranwow/dalaran-wow#5315]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 
When u kill prince Tortheldrin u are not able to loot the chest that is on the floor below.

**Expected behaviour**: 
U should be able to loot the chest/box and recive quest item Gauntlet of Gordok Might. Some sources also state that the chest should be behind prince Tortheldrin and not on the lower level floor.

**Steps to reproduce the problem**:
1. Acquire Gordok ogre suit and use it. Proceed to talk with Captain Kromcrush in Dire maul and get the quest "Unfinished Gordok Business"
2. Find prince Tortheldrin and kill him
3. Jump on the lower floor below him and in the corner try to loot the chest/box with quest item.

**Include proofs for this behaviour**
http://www.wowhead.com/quest=1318/unfinished-gordok-business#comments

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Dungeon = Dire Maul North
gauntlet-of-gordok-might =18336 
the-princes-chest = 179545 
prince-tortheldrin = 11486
captain-kromcrush = 14325
unfinished-gordok-business = 1318

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Duplicate of https://github.com/dalaranwow/dalaran-wow/issues/2503. I fixed everything (also the gameobject issue) in https://github.com/dalaranwow/dalaran-wow/issues/2503. I will reapply the fix on PTR soon.
